src/javascript/app_2/App/Components/Elements/Calendar/calendar.jsx

```diff
@@ -34,6 +36,37 @@ class Calendar extends React.PureComponent {
         });
     };
 
+    onMouseOver = (event) => {
+        const target = event.currentTarget;
+
+        if (!target.classList.contains('calendar__cell--disabled') && !target.classList.contains('calendar__cell--hover')) {
+            target.className += ' calendar__cell--hover';
+            this.setState({
+                hovered_date : target.getAttribute('data-date'),
+                duration_date: target.getAttribute('data-duration'),
+            });
+        }
+    };
+
+    onMouseLeave = (event) => {
+        const target = event.currentTarget;
+
+        if (target.classList.contains('calendar__cell--hover')) {
+            const moment_today = toMoment().startOf('day');
+            const moment_selected = toMoment(this.state.selected_date);
+            const default_duration = moment_selected.diff(moment_today, 'days');
```

Comment:
would be good to use `daysFromTodayTo` from `Utils/Date` to get the difference 😄